### DISCUSSION TITLE
🙏 Self-Hosted Support

### DISCUSSION THREAD

**Discussion by DmitryPopov** [wanderer-industries/wanderer#4] [Q&A]
Ask the community for help around your Wanderer CE instance

Join Our [Discord community](https://discord.gg/cafERvDD2k) to get answers on your questions quicker from other community members!

**Comment by olf-folks** [comment]
when i go to /admin it just redirects me to the /welcome
community-edition-wanderer-1     | 07:42:20.577 application=phoenix module=Phoenix.Logger function=phoenix_endpoint_stop/4 line=231 request_id=F_aVXpL-Pfn9mcEAAABj [info] Sent 200 in 9ms
community-edition-wanderer-1     | 07:43:55.000 application=phoenix module=Phoenix.Logger function=phoenix_endpoint_start/4 line=217 request_id=F_aVdI-PIdjAyLAAAACD [info] GET /admin
community-edition-wanderer-1     | 07:43:55.001 application=phoenix module=Phoenix.Logger function=phoenix_endpoint_stop/4 line=231 request_id=F_aVdI-PIdjAyLAAAACD [info] Sent 302 in 432µs
community-edition-wanderer-1     | 07:43:55.012 application=phoenix module=Phoenix.Logger function=phoenix_endpoint_start/4 line=217 request_id=F_aVdJBB53TAyLAAAADD [info] GET /welcome

**Reply by DmitryPopov** [reply]
You should have at least 1 character registered to access Admin page. Updated instruction - https://github.com/wanderer-industries/community-edition/blob/main/README.md#authorize-your-first-character

**Comment by aevans1987** [comment]
wanderer-1     | 03:09:34.215 application=phoenix module=Phoenix.Socket.Transport function=check_origin/5 line=344 [error] Could not check origin for Phoenix.Socket transport.
wanderer-1     |
wanderer-1     | Origin of the request: https://sub.domain.tld
wanderer-1     |
wanderer-1     | This happens when you are attempting a socket connection to
wanderer-1     | a different host than the one configured in your config/
wanderer-1     | files. For example, in development the host is configured
wanderer-1     | to "localhost" but you may be trying to access it from
wanderer-1     | "127.0.0.1". To fix this issue, you may either:
wanderer-1     |
wanderer-1     |   1. update [url: [host: ...]] to your actual host in the
wanderer-1     |      config file for your current environment (recommended)
wanderer-1     |
wanderer-1     |   2. pass the :check_origin option when configuring your
wanderer-1     |      endpoint or when configuring the transport in your
wanderer-1     |      UserSocket module, explicitly outlining which origins
wanderer-1     |      are allowed:
wanderer-1     |
wanderer-1     |         check_origin: ["https://example.com",
wanderer-1     |                        "//another.com:888", "//other.com"]

I am getting this when i try to load the site and establish a connection after authorizing my first character. I have set the BASE_URL in the .env file and have confirmed it is correct. The page loads and gives me a Re-establishing connection toast and just never goes away or loads.

**Reply by DmitryPopov** [reply]
@aevans1987  BASE_URL was renamed to WEB_APP_URL, please check the updated README and wanderer-conf.env file here
https://github.com/wanderer-industries/community-edition/tree/main

**Comment by hjheins** [comment]
Hi,
I just tried to install your software on a little arm server I have here, but unfortunately with docker thats a bit of a bear to do (no arm support currently). A full manual install seems to be not an option, so thats a bit of a shame. I think you software would generally work very well on arm as well. Could there be an option in the future to have a "on iron" install?  thanks.

**Comment by aureness** [comment]
All I got after `docker compose -f docker-compose.yml up -d` is recurrent error in logs

```
Loading wanderer_app..
Starting dependencies..
Starting repos..
** (MatchError) no match of right hand side value: {:error, {:error, %DBConnection.ConnectionError{message: "connection not available and request was dropped from queue after 10997ms. This means requests are coming in and your connection pool cannot serve them fast enough. You can address this by:\n\n  1. Ensuring your database is available and that you can connect to it\n  2. Tracking down slow queries and making sure they are running fast enough\n  3. Increasing the pool_size (although this increases resource consumption)\n  4. Allowing requests to wait longer by increasing :queue_target and :queue_interval\n\nSee DBConnection.start_link/2 for more information\n", severity: :error, reason: :queue_timeout}}}
    (wanderer_app 1.92.0) lib/wanderer_app/release.ex:16: anonymous fn/2 in WandererApp.Release.createdb/1
    (elixir 1.17.2) lib/enum.ex:2531: Enum."-reduce/3-lists^foldl/2-0-"/3
    (wanderer_app 1.92.0) lib/wanderer_app/release.ex:15: WandererApp.Release.createdb/1
    nofile:1: (file)
    (stdlib 6.0.1) erl_eval.erl:904: :erl_eval.do_apply/7
```
